pkg/broker/broker_test.go

```diff
@@ -64,3 +65,176 @@ func ExampleBrokerService_IsEnabled() {
 	// Output: true
 	// false
 }
+
+func ExampleBrokerService_ServiceDefinition() {
```

Comment:
Are these `Example...` funcs to demonstrate usage?

Reply:
Yes, they're the executable examples that show up in godoc but also count towards unit test coverage. I wanted to document the common functions as examples to show how they interact with Viper.